erpnext/projects/doctype/task/task.js

```diff
@@ -5,7 +5,7 @@ frappe.provide("erpnext.projects");
 
 frappe.ui.form.on("Task", {
 	setup: function (frm) {
-		frm.set_query("project", function () {
+		frm.set_query("default_project", function () {
```

Comment:
this set_query doesn't make sense as default project is read_only. Move this to the child table maybe?